app/views/admin/frameworks/show.html.haml

```diff
@@ -7,11 +7,11 @@
       %h2#page-actions-title.govuk-heading-s{"aria-label" => "Page actions"} Actions
       %ul.govuk-page-actions--actions
         %li.govuk-page-actions--action
-          = link_to 'Edit definition', edit_admin_framework_path(@framework.id)
+          = link_to 'Edit definition', edit_admin_framework_path(@framework)
```

Comment:
Might be worth explaining why the `id` call is unnecessary in the commit message (I'm unsure on first glance).